assets/style.css

```diff
@@ -0,0 +1,177 @@
+body {
+	margin:0;
+}
+.nav {
+   margin: 0;
+}
+
+.nav li {
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.logo {
+  height: 50px;
+  width: 100px;
+  vertical-align: middle;
+}
+
+.menu_bar {
+  margin:0;
+  position: relative;
+  right: 20px;
+}
+
+.menu_bar img{
+  height: 30px;
+  width: 30px;
+  vertical-align: middle;
+  margin: 5px;
+}
+
+.nav__searchbar {
+  width: 50%;
+  margin-right: 0;
+  padding-right: 0;
+}
+
+.nav__searchbar input {
+  width: 100%;
+  height: 30px;
+}
+
+.nav__searchbar-button {
+  margin: 0;
+  padding: 0;
+  padding-left: 0px;
+  margin-left: 0px;
+}
+
+.nav__searchbar-button input {
+  height:36px;
+  width:60px;
+  background:#EEEEEE;
+  background-image:url(https://img.icons8.com/ios/50/000000/search.png);
+  background-position:center;
+  background-size:20px;
+  background-repeat: no-repeat;
+  border-radius:5%;
+  cursor:pointer;
+  vertical-align: middle;
+  }
+
+.icons {
+  height: 30px;
+  width: 30px;
+  vertical-align: middle;
+  margin: 5px;
+  position: relative;
+  left: 90px;
```

Comment:
in order to move icons to the right on the navbar, you should push them away from the input button, in your case list items and anchors are not in the same position. Also, add some display properties to the anchor.

Reply:
In order to achieve this, we placed an unnordered list inside a div, wich was the one that we pushed away from the search button, also we make sure, that the li, and the anchors were in the same position.